python/vault/fybrik_python_vault/__init__.py

```diff
@@ -0,0 +1,98 @@
+#
+# Copyright 2022 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+#
+import requests
+from fybrik_python_logging import logger, Error, DataSetID, ForUser
+
+def test_func():
```

Comment:
Seems like test_func() doesn't do anything, right?

Reply:
right, I removed it.